src/_includes/site_navigation.html

```diff
@@ -0,0 +1,34 @@
+{% assign sections = site.pages | site_sections %}
+{% assign parent_page = site.pages | parent_of: page %}
+
+<aside class="related">
+  <nav aria-label="Site navigation">
+    <ul>
+      {% for section in sections %}
+          <li class="related__section-link">
+            <a href="{{ section.url | relative_url }}" {% if section.url == page.url %}class="current" aria-current="page"{% endif %}>{{ section.title }}</a>
+              {% assign grouped_related_pages = site.pages | children_of: section | group_by: "related_order" | sort: "name" %}
+              {% if grouped_related_pages.size > 0 %}
+                <label class="visually-hidden" for="section-{{ forloop.index }}">Toggle page links in this section</label>
+                <input class="related__expand-toggle arrow" type="checkbox" name="section-{{ forloop.index }}" id="section-{{ forloop.index }}"
```

Comment:
Not a blocker, but it would be good to make a point of user researching the checkbox toggle method used here at a future date as I'm not sure it's a widely used approach compared to button toggles with JavaScript. There may be a good reason why it isn't widely used.

Reply:
good idea, i wanted to try and start with a JS-free implementation, but I have seen examples with both - is there a good place to track this?

Reply:
There's this from Inclusive Components [toggle-button](https://inclusive-components.design/toggle-button/) - that makes a good point that:

> The trouble with using form elements is their longstanding association with the collection of data

Reply:
Thanks Calum, I have created a separate issue for this, as will need to think about how we want to add JavaScript. So makes sense to work on this in a later PR?